kolibri/plugins/device/assets/src/views/ManageContentPage/FilteredChannelListContainer.vue

```diff
@@ -190,53 +195,31 @@
 <style lang="scss" scoped>
 
   .top-panel {
-    display: flex;
-    flex-wrap: wrap;
-    align-items: flex-start;
     margin-bottom: 24px;
-  }
 
-  .count-msg {
-    flex-grow: 1;
-    max-width: 200px;
-    margin: 0;
+    .fclc-sm & {
```

Comment:
I am not able to guess what `fclc` means. Would prefer that we use more verbose class names for code maintainability

Reply:
Ah, `FilteredChannelListContainer` - got it.

Agreed that root component nodes can be difficult because the scope namespace is shared with the parent.